auslib/web/public/client.py

```diff
@@ -93,7 +93,31 @@ def getQueryFromURL(url):
     return query
 
 
+def extract_query_version(request_url):
+    version = 0
+    pattern = '^.*/update/([1-6])/.*\.xml$'
```

Comment:
It's probably good to futureproof this could by accepting any integer in this field.